src/Read_Forcing.f90

```diff
@@ -6,84 +6,77 @@ SUBROUTINE Read_Forcing
 ! 
 IMPLICIT NONE
 !
-integer:: nc,ncell,nnd,no_flow,no_heat,nr,nrec_flow,nrec_heat
+integer:: nc,ncell,nnd,no_flow,no_heat,nr
 real:: Q_avg
 
 
 no_flow=0
 no_heat=0
 do nr=1,nreach
-  do nc=1,no_cells(nr)-1 
-    no_flow=no_flow+1
-    no_heat=no_heat+1
+  do nc=1,no_cells(nr)-1 
+    no_flow=no_flow+1
+    no_heat=no_heat+1
 !
```

Comment:
NREC_FLOW and NREC_+HEAT are no longer necessary. They used in the previous version to identify locations in the DIRECT ACCESS file